json-endpoints/class.wpcom-json-api-list-roles-endpoint.php

```diff
@@ -86,6 +86,8 @@ static function role_sort( $a, $b ) {
 
 	// /sites/%s/roles/ -> $blog_id
 	function callback( $path = '', $blog_id = 0 ) {
+		require_lib( 'wpforteams' );
```

Comment:
@annemirasol probably this library isn't available in Jetpack, I think a cleaner solution could be to add a `do_filter` before `return` of the endpoint and hook to it from within wpcom.